data-prepper-plugins/mongodb/src/main/java/org/opensearch/dataprepper/plugins/mongo/coordination/state/ExportProgressState.java

```diff
@@ -21,6 +21,9 @@ public class ExportProgressState {
     @JsonProperty("status")
     private String status;
 
+    @JsonProperty("lastEndDocId")
+    private Object lastEndDocId;
```

Comment:
This is just tracking the export progress for creating the partitions?

Reply:
Yes. If the partitions creation has been interrupted due to credential rotation, this will be the resume point.